go.mod

```diff
@@ -5,18 +5,21 @@ go 1.18
 require (
 	code.cloudfoundry.org/jsonry v1.1.3
 	github.com/Pallinder/go-randomdata v1.2.0
-	github.com/cloudfoundry/cloud-service-broker v0.10.1
```

Comment:
Typically we don't update this.

Reply:
Thank you very much, @blgm. I made a mistake adding this modification. I'll change it now.

As for splitting this PR. It sounds good to me! I'll start now. I'm sorry I mixed up concepts